CodeU_Assignement_5/src/codeu_assignement_5/KemUnknownLanguage.java

```diff
@@ -0,0 +1,145 @@
+package codeu_assignement_5;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ *
+ * @author Karine
+ */
+public class KemUnknownLanguage {
+    /*
+     * Input: a dictionary (a list of words in lexicographic order) of all words in an unknown/invented language, 
+    *  Output: alphabet (an ordered list of characters) of that language, letters we cannot decide are appended in the end.
+    */
+    public List<Character> getAlphabeit(String[] alienWords) {
+        if ((alienWords != null) && (alienWords.length > 0)) {
+           return extractAlphabeitFromWords(alienWords);
+        }
+        
+        return null;
+    }
```

Comment:
Now I'm confused. Which version is better?

if(alienWords == null || alienWords.length == 0) return null;
=== VS. ===
List<Character> alphabet = null; if ((alienWords != null) && (alienWords.length > 0)) { .... } return alphabet;

Or do you have another idea?